config/routes.rb

```diff
@@ -1,8 +1,12 @@
 Bars::Application.routes.draw do
+  devise_for :admins
   devise_for :users, controllers: { sessions: "users/sessions" }
 
   root to: "bars#index"
   resources :bars do
     resources :reviews, only: [:new, :create, :edit, :update, :destroy]
   end
+  resources :admins, only: [:index, :create, :destroy]
+  get "make_admin", to: "admins#make_admin"
```

Comment:
If you find yourself creating paths in your Rails application that aren't named the following, you are working against the framework conventions: new create edit update index show destroy.